### PR TITLE
Update mutability tag to link Nedbat's talk

### LINKED ISSUE #2545: Add a link to Nedbat's talk on mutability to the mutability tag
Nedbat's video is linked so often; I think it would be nice to just include it in the response to !mutability

### PR DESCRIPTION
**Pull request by jahrules**
Closes #2545

### CODE REVIEW

File: bot/resources/tags/.pep8.md.swp
Comment (etrotta) [inline bot/resources/tags/.pep8.md.swp:1]: I don't think that this file should be included?

File: bot/resources/tags/mutability.md
Comment (etrotta) [inline bot/resources/tags/mutability.md:1]: There shouldn't be a space between the `]` and `(`

**Review by etrotta — DISMISSED** [review]
Wait until the Issue is actually approved before working on it

File: bot/resources/tags/mutability.md
Comment (jahrules) [reply bot/resources/tags/mutability.md:1]: fixed.

File: bot/resources/tags/.pep8.md.swp
Comment (jahrules) [reply bot/resources/tags/.pep8.md.swp:1]: fixed. I (wrongly) assumed that swp files were in .gitignore

File: bot/resources/tags/mutability.md [hunk 1]
```diff
@@ -37,3 +37,6 @@ Mutable data types like `list`, on the other hand, can be changed in-place:
 ```
 
 Other examples of mutable data types in Python are `dict` and `set`. Instances of user-defined classes are also mutable.
+
+For an in-depth guide on mutability see:
+[Nedbat's video on mutability](https://youtu.be/_AEJHKGk9ns/)
```
Comment (wookie184) [inline bot/resources/tags/mutability.md:42]:
```suggestion
For an in-depth guide on mutability see [Ned Batchelder's video on names and values](https://youtu.be/_AEJHKGk9ns/).
```

**Review by shenanigansd — APPROVED** [review]

**Pull request merged**